README.md

```diff
@@ -62,6 +67,14 @@ The following properties are used if your app has an API:
 
 This is the list of API versions your app can use. Since `v1` is the default, you'll usually want at least that one defined.
 
+#### app_id.api.subItems
+
+If your API consists of multiple APIs you can list them in here it has the same signature as `{app_id.api}`.
```

Comment:
Grammatical change:
"If your API consists of multiple APIs, you can list them here. Each has the same signature as `{app_id.api}`."

Reply:
Thank you! Should be fixed.